packages/web3-utils/test/unit/random.test.ts

```diff
@@ -24,6 +24,7 @@ describe('random library tests', () => {
 		describe('valid cases', () => {
 			it.each(randomHexData)('%s', input => {
 				const hexResult = randomHex(input);
+				// eslint-disable-next-line deprecation/deprecation
```

Comment:
What was the linter erroring about here?

Reply:
`isHexStrict` is marked [here](https://github.com/ChainSafe/web3.js/blob/f4cae280db3298b9e8e501a995083031c507dcf9/packages/web3-utils/src/validation.ts#L34) as deprecated and linter was throwing an error.